pkg/sbom/spdx/marshal.go

```diff
@@ -148,11 +149,23 @@ func (m *Marshaler) Marshal(r types.Report) (*spdx.Document, error) {
 			if err != nil {
 				return nil, xerrors.Errorf("package file error: %w", err)
 			}
-			spdxFiles = append(spdxFiles, files...)
-			for _, file := range files {
-				relationShips = append(relationShips,
-					relationShip(spdxPackage.PackageSPDXIdentifier, file.FileSPDXIdentifier, RelationShipContains),
-				)
+			if files != nil {
```

Comment:
I prefer less indent.

```
			files, err := m.pkgFiles(pkg)
			if err != nil {
				return nil, xerrors.Errorf("package file error: %w", err)
			} else if files == nil {
				continue
			}

Reply:
Changed in https://github.com/aquasecurity/trivy/pull/5533/commits/f7770a7c5caa80872c9347ee327cba738e709b75